apiary.apib

```diff
@@ -428,7 +428,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### October 2023
 - We added the `calls.list` and `calls.info` endpoint.
-- We added the `callOutcome.list` endpoint.
+- We added the `callOutcomes.list` endpoint.
+- We added the `calls.complete` endpoint.
```

Comment:
Q: Do we "complete" a call, or do we "end" a call?

Reply:
> Q: Do we "complete" a call, or do we "end" or maybe "finish" a call?
> 
> Not sure the term "complete" is the best fit here.

🤔 aren't those synonyms?
I think when it was introduced one of the goals was also to have consistency between all the events endpoints (calls.complete, tasks.complete, meetings.complete) even if the term may not be the perfect for each 🤷‍♂️